docs/UserGuide.adoc

```diff
@@ -111,6 +111,29 @@ Deletes the 2nd person in the address book.
 `delete 1` +
 Deletes the 1st person in the results of the `find` command.
 
+
+
+
+== Edit a person's Email : `edit_email`
+
+Displays the non-private details of the specified person. +
+Format: `edit INDEX e/KEYWORD`
+
+****
+Edit the email of the person at the specified `INDEX`.
+The index refers to the index number shown in the most recent listing.
+****
+
+Examples:
+
+* `list` +
+`edit 2 e/newemail@gmail.com` +
```

Comment:
Doesn't match the command you have in the code!
Be careful of such inconsistencies. In the project, these can cost you points when your tester is unable to test some command.